src/BasePlatform.ts

```diff
@@ -236,7 +236,7 @@ export default abstract class BasePlatform {
      * @returns {boolean} whether the platform requires URL previews in tooltips
      */
     public needsUrlTooltips(): boolean {
-        return false;
+        return true;
```

Comment:
I presume this is an artifact of your local testing?

Reply:
Argh, sorry. This is what happens when you try to fix your mess shortly before going into a meeting. Removed it.